src/ucp/tag/rndv.h

```diff
@@ -28,7 +23,6 @@ typedef struct {
     ucp_request_hdr_t         sreq;     /* send request on the rndv initiator side */
     uint64_t                  address;  /* holds the address of the data buffer on the sender's side */
     size_t                    size;     /* size of the data for sending */
-    uint16_t                  flags;
```

Comment:
How do we check on receiver whether we have rkeys or not now?

Reply:
according to md_map in the packed rkey

Reply:
but we send no rkey in case of SW rndv, non-contig RNDV, etc

Reply:
in this case, we can tell it by address==NULL

Reply:
BTW, for SW rndv, we do send rkeys (if the send buffer is contig but large, for example)

Reply:
aa, right, we check the address. Yes, SW rndv may carry the rkey for large messages, but in other cases it won't.